packages/javascript/bh-shared-ui/src/views/TierManagement/Save/SelectorForm/SelectorForm.test.tsx

```
@@ -264,7 +263,7 @@ describe('Selector Form', () => {
         render(
             <Routes>
                 <Route path={'/'} element={<SelectorForm />} />;
-                <Route path={'/tier-management/save/tier/:tierId/selector/:selectorId'} element={<SelectorForm />} />;
+                <Route path={'/tier-management/save/tier/:tierId/selector/:selectorId'} element={<SelectorForm />} />
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)